extensions/disruptor/runtime/pom.xml

```diff
@@ -87,6 +92,8 @@
                             <version>${quarkus.version}</version>
                         </path>
                     </annotationProcessorPaths>
+                    <source>9</source>
+                    <target>9</target>
```

Comment:
We still support target 1.8. If this is really required, it should be properly explained in the limitations.adoc, IMO. Why is this necessary, BTW?

Reply:
Already spotted by @davsclaus